pkg/tools/etcd_helper_test.go

```diff
@@ -685,7 +687,7 @@ func TestGuaranteedUpdate_CreateCollision(t *testing.T) {
 
 func TestGetEtcdVersion_ValidVersion(t *testing.T) {
 	testServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprint(w, "etcd 2.0.9")
+		fmt.Fprint(w, etcdVersion209)
```

Comment:
To be clear, in my original comment I meant to suggest that that your constant be named something like validEtcdVersion, so that it is only necessary to update it at one place near the head of the file when we change etcd versions, and not in multiple places scattered through the code.  With this change you need to change it in three places instead of two or one.  It least the compiler should catch it like this, but I still think that it's better to take the version number out of the constant name.

As a second comment, will we want to support multiple etcd versions?  If so, perhaps an array of supported versions is more appropriate.